### ISSUE TITLE
[BUG] Absolute PERSIST_PATH is used relative to cwd

### ISSUE THREAD

**Issue by alexrashed** [etspaceman/kinesis-mock#554]
**Describe the bug**
With the latest releases, the `PERSIST_PATH` is always used as being relative to the current working directory (even though it starts with a `/`).

**To Reproduce**
```
#!/bin/bash
# start with absolute path
SHOULD_PERSIST_DATA=true PERSIST_PATH="/tmp/state/kinesis" PERSIST_FILE_NAME=state.json PERSIST_INTERVAL=5s LOG_LEVEL=DEBUG node main.js &

# save the PID
KINESIS_PID=$!

# create some data
aws --endpoint-url="http://localhost:4568/" kinesis create-stream --stream-name test

# make sure everything is persisted
sleep 10

# send the interrupt
kill -2 $KINESIS_PID

# the absolute directory does not exist
ls /tmp/state/kinesis

# the relative one does
ls tmp/state/kinesis
```

**Expected behavior**
When a `PERSIST_PATH` with a leading slash is set, the path should be used as an absolute path (and not relative to the current working directory).

**Additional context**
I was facing this issue when testing our persistence feature for LocalStack, based on https://github.com/localstack/localstack/pull/8790.
A workaround was implemented for https://github.com/localstack/localstack/pull/8790 with https://github.com/localstack/localstack/pull/8790/commits/e5944c2b3cc3cd8e1207f0f8a3a8d5a62c102f85

**Comment by etspaceman** [comment]
Good call. I have a fix ready for this that is building.

**Comment by etspaceman** [comment]
Released in 0.4.3
